vg/tee.go

```diff
@@ -0,0 +1,139 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vg
+
+import (
+	"image"
+	"image/color"
+)
+
+// MultiCanvas creates a canvas that duplicates its drawing operations to all
+// the provided canvases, similar to the Unix tee(1) command.
+//
+// Each drawing operation is sent to each listed canvas, one at a time.
+func MultiCanvas(cs ...Canvas) Canvas {
+	return teeCanvas{cs}
+}
+
+type teeCanvas struct {
+	cs []Canvas
+}
+
+// SetLineWidth sets the width of stroked paths.
+// If the width is not positive then stroked lines
+// are not drawn.
+//
+// The initial line width is 1 point.
```

Comment:
For these comments on "The initial ...", I would rather say that the initial state depends on the underlying `Canvas`.

Reply:
I removed those.